example/hands-on/services/bucky_hands_on/pc/scenarios/linkstatus/github_top.yml

```diff
@@ -0,0 +1,16 @@
+# Describe for this test suite
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.